src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -2399,4 +2399,13 @@ Uninstall the following packages: {0}?</value>
   <data name="ResetCPythonButtonToolTip" xml:space="preserve">
     <value>Resets CPython environment by reloading modules.</value>
   </data>
+  <data name="InstalledPackagesExpanderName" xml:space="preserve">
+    <value>Installed Packages</value>
```

Comment:
Can you look for same string within Dynamo code base? I think we already have this string.
![image](https://user-images.githubusercontent.com/3942418/121056441-0e4c9b00-c773-11eb-92dc-514980fa4d82.png)

Reply:
sure, will look.